tsdb/db_test.go

```diff
@@ -517,7 +516,6 @@ func TestDB_Snapshot_ChunksOutsideOfCompactedRange(t *testing.T) {
 		testutil.Ok(t, err)
 	}
 	testutil.Ok(t, app.Commit())
-	testutil.Ok(t, app.Rollback())
```

Comment:
Can you leave those with `NotOk` then? (:

Reply:
That seems unrelevant here but I can add a test for this.